### DISCUSSION TITLE
Cannot figure out Python Syntax Highlighting

### DISCUSSION THREAD

**Discussion by jkbrookes2** [graphql/graphiql#3265] [Q&A / Support]
I cannot for the life of me figure out how to use the syntax highlighting for python.  It is not in the documentation and I've taken a look at the python grammar file here but all I've gotten is that """#graphql will enable the highlighting BUT that is also inserting #graphql into my query string.

**Comment by acao** [comment]
Hello @jkbrookes2 ! I wrote some tests for this last night and plan to be updating the readme

https://github.com/graphql/graphiql/blob/main/packages/vscode-graphql-syntax/tests/__fixtures__/test.py

I wish i had a better answer for you, but these delimiters are based on what the python clients are using. If you'd like to propose another syntax, please suggest one!

**Comment by woprandi** [comment]
I'm also unable to make it works inside python file. No problem for `.graphql` file. No logs from langage server
